.circleci/config.yml

```diff
@@ -13,7 +13,7 @@ jobs:
       - node/install-packages:
           pkg-manager: npm
       - run:
-          command: npm run test
+          command: npm run --passWithNoTests
```

Comment:
Also, why is this changed?

Reply:
because I removed the bogus 2+2=4 test. Since I removed it there's no more tests so I have to tell it to pass without tests othewise it will complain that its looking for tests and theres none

Reply:
Ahh I see. Thanks for clarifying!